ui/Screen/IdentityCreation.svelte

```diff
@@ -0,0 +1,49 @@
+<script>
+  import { pop, replace } from "svelte-spa-router";
+
+  import { showNotification } from "../store/notification.js";
+  import { ModalLayout } from "../DesignSystem/Component";
+  import { Button } from "../DesignSystem/Primitive";
+  import IdentityCreationForm from "./Identity/IdentityCreationForm.svelte";
+  import IdentityCreationSuccess from "./Identity/IdentityCreationSuccess.svelte";
+
+  const steps = {
```

Comment:
@xla @rudolfs @MeBrei this is one way to do js "enums"; how do you feel about it? It makes the markup more readable but ofc isn't as pretty as _real_ enums 😜

Reply:
Definitely an improvement over just raw numbers, or ungrouped consts. This is friendlier for the reader to follow what is actually behind the steps.